client/src/components/overview/Overview.jsx

```diff
@@ -1,13 +1,51 @@
 import React, { useState, useEffect } from 'react';
 import fetcher from '../../fetchers';
 
+///// SUB-COMPONENETS /////
+import Search from './sub-comps/Search.jsx';
+import Gallery from './sub-comps/Gallery.jsx';
+import ProdInfo from './sub-comps/ProdInfo.jsx';
+import StyleSelect from './sub-comps/StyleSelect.jsx';
+import AddCart from './sub-comps/AddCart.jsx';
+import Description from './sub-comps/Description.jsx';
+
+
 export default function Overview({
-  feature
+  feature,
+  setFeature
 }) {
 
+  ///// STATE DATA /////
+  const [prodStyle, setStyle] = useState({});
+  const [description, setDisplay] = useState(true);
+
+  console.log('feature: ', feature);
+  console.log('style: ', prodStyle);
+
+
+
+  ///// INITIALIZATION /////
+  useEffect(() => {
+    fetcher.overview.getStylesById(40344)
+      .then(result => setStyle(result.data))
+      .catch(err => console.log('err initial styles fetch: ', err));
+  }, []);
```

Comment:
Maybe to put feature into the useEffect monitor [], so it will update the style when feature changed.